python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -776,7 +779,10 @@ def _tree_node_agg(dfs, gb_cols, split_out, dropna, sort, sep):
     )
 
     # Don't include the last aggregation in the column names
-    gb.columns = [_make_name(*name[:-1], sep=sep) for name in gb.columns]
+    gb.columns = [
+        _make_name(name[:-1] if isinstance(name, tuple) else name, sep=sep)
```

Comment:
If `_make_name` always returned a `list`, the conditional here wouldn't be needed.

Reply:
Could you elaborate on this? Is the thought that we would have `_make_name` do something like this?

```python
def _make_name(columns, sep="_"):
    return [
        col
        if isinstance(col, str)
        else sep.join(name for name in col if name != "")
        for col in columns
    ]
```

Reply:
I was thinking of just simply...

```
if isinstance(col_name, str):
    return [col_name]
return sep.join(name for name in col_name if name != "")
```

Reply:
Ah okay I see what you mean - I think this might be more refactoring to the code than it's worth to avoid the conditional, as there are several places where we assume the columns are strings and would need to either add or change variable assignment to unpack the lists.